### PR TITLE
Add churn command

### PR DESCRIPTION
**Pull request by Buddybenj**

### CODE REVIEW

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -1,6829 +1,6859 @@
...
-                                  "confirm-missing-payment-id <1|0>\n "
-                                  "ask-password <1|0>\n "
-                                  "unit <monero|millinero|micronero|nanonero|piconero>\n "
-                                  "  Set the default monero (sub-)unit.\n "
-                                  "min-outputs-count [n]\n "
-                                  "  Try to keep at least that many outputs of value at least min-outputs-value.\n "
-                                  "min-outputs-value [n]\n "
-                                  "  Try to keep at least min-outputs-count outputs of at least that value.\n "
-                                  "merge-destinations <1|0>\n "
-                                  "  Whether to merge multiple payments to the same destination address.\
```
Comment (leonklingele) [inline src/simplewallet/simplewallet.cpp]: Oops, looks like you switched to Windows file endings. Should fix your editor.

File: src/simplewallet/simplewallet.h [hunk 2]
```diff
@@ -1,342 +1,343 @@
...
+    bool transfer_main(int transfer_type, const std::vector<std::string> &args);
+    bool transfer(const std::vector<std::string> &args);
+    bool transfer_new(const std::vector<std::string> &args);
+    bool locked_transfer(const std::vector<std::string> &args);
+    bool sweep_main(uint64_t below, const std::vector<std::string> &args);
+    bool sweep_all(const std::vector<std::string> &args);
+    bool sweep_below(const std::vector<std::string> &args);
+    bool sweep_single(const std::vector<std::string> &args);
+    bool sweep_unmixable(const std::vector<std::string> &args);
+	bool churn(const std::vector<std::string> &args);
```
Comment (leonklingele) [inline src/simplewallet/simplewallet.h]: Indent using spaces instead of tabs.

**Comment by stoffu** [comment]
The proposed `churn` command seems like a simple alias to `sweep_all <one's_own_address>`. I don't think this is a particularly good solution to the longstanding question of how churning should be done effectively. For one thing, having all the outputs consolidated into one single output due to sweeping all to self would decrease the usability since all the funds get locked for every outgoing transfer.

We seem to largely agree on the need for churning, but we don't know yet what would actually be the effective solution. I'd rather reserve this `churn` command for a better solution that will hopefully be found in the future. In the meantime, users who are particularly concerned about the highest degree of obfuscation can just do `sweep_all <own_address>` manually, which might be a little tedious, but still doable.

**Comment by moneromooo-monero** [comment]
Agreed with stoffu. The command is good to have, but it should do what it says on the tin first.

**Comment by jtgrassie** [comment]
Also, why all the tab/space/line-ending changes?

**Comment by Buddybenj** [comment]
Closing based on stoffu's comments.

**Pull request closed without merging**